amixr/mezmo/README.md

```diff
@@ -0,0 +1,26 @@
+# Agent Check: Mezmo Telemetry Pipeline
+
+## Overview
+
+Mezmo Telemetry Pipeline empowers teams to ingest, transform, and route telemetry data in real-time. You can ingest telemetry data from various sources via our open platform, apply transformation and processing logic, then route to Datadog.
```

Comment:
The overview section should include the following:

1. What does Mezmo do in 1-2 sentences?
2. What will customers get from this integration?
3. What types of data does this send in and the value of the data?